src/plotman/csv_exporter.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import sys
+from dateutil.parser import parse as parse_date
+from plotman.log_parser import PlotLogParser
+from plotman.plotinfo import PlotInfo
+
+def export(logfilenames, save_to = None):
+    if save_to is None:
+        send_to_stdout(logfilenames)
+    else:
+        save_to_file(logfilenames, save_to)
+
+def save_to_file(logfilenames, filename: str):
+    with open(filename, 'w') as file:
+        generate(logfilenames, file)
+
+def send_to_stdout(logfilenames):
+    generate(logfilenames, sys.stdout)
+
+def header(writer):
+    writer.writerow([
+        'Plot ID', 
+        'Started at',
+        'Date',
+        'Size',
+        'Buffer',
+        'Buckets',
+        'Threads',
+        'Tmp dir 1',
+        'Tmp dir 2',
+        'Phase 1 duration (raw)',
+        'Phase 1 duration',
+        'Phase 1 duration (minutes)',
+        'Phase 1 duration (hours)',
+        'Phase 2 duration (raw)',
+        'Phase 2 duration',
+        'Phase 2 duration (minutes)',
+        'Phase 2 duration (hours)',
+        'Phase 3 duration (raw)',
+        'Phase 3 duration',
+        'Phase 3 duration (minutes)',
+        'Phase 3 duration (hours)',
+        'Phase 4 duration (raw)',
+        'Phase 4 duration',
+        'Phase 4 duration (minutes)',
+        'Phase 4 duration (hours)',
+        'Total time (raw)',
+        'Total time',
+        'Total time (minutes)',
+        'Total time (hours)',
+        'Copy time (raw)',
+        'Copy time',
+        'Copy time (minutes)',
+        'Copy time (hours)',
+        'Filename'
+    ])
+
+def parse_logs(logfilenames):
+    parser = PlotLogParser()
+    result = []
+
+    for filename in logfilenames:
+        info = parser.parse(filename)
+
+        if not info.is_empty():
+            result.append(info)
+
+    result.sort(key=log_sort_key)
+    return result
+
+def log_sort_key(element: PlotInfo):
+    return parse_date(element.started_at).replace(microsecond=0).isoformat()
+
+def generate(logfilenames, out):
+    writer = csv.writer(out)
+    header(writer)
+    logs = parse_logs(logfilenames)
+
+    for info in logs:
+        writer.writerow([
+            info.plot_id,
+            info.started_at,
+            parse_date(info.started_at).strftime("%d %b %Y"),
```

Comment:
Seems like a well formed standard like ISO 8601 or RFC 3339 would be better than a locale specific time here?

Reply:
Yeah, makes sense. I"ll adjust